CONTRIBUTING.md

```diff
@@ -1,27 +1,136 @@
-# Express Collaborator Guide
+# Contributing to Express Ecosystem
 
-## Website Issues
+Thank you for your interest in contributing to the Express ecosystem! This document outlines
+how you can contribute to the Express project, including reporting issues, submitting code,
+and participating in discussions.
 
-Open issues for the expressjs.com website in https://github.com/expressjs/expressjs.com.
+Express has a open governance model. For information about the governance of the Express project,
+see the [GOVERNANCE.md](https://github.com/expressjs/discussions/blob/HEAD/docs/GOVERNANCE.md)
+document.
 
-For issues in other Express managed repos (everything in `expressjs`, `pillarjs` or `jshttp` other than `expressjs/express`), be sure to check their contributing guide and open issues and PRs in the appropriate repository.
+## Code of Conduct
 
-## PRs and Code contributions
+Everyone who participates in this project, either as a user or a contributor, is obliged
+to follow the project [Code of Conduct](https://github.com/expressjs/.github/blob/HEAD/CODE_OF_CONDUCT.md).
 
```

Comment:
[suggestion]: It would be helpful to mention the different types of contributions users can make to the project. It would demystify the idea that code is the only way they can contribute to the project.